deploy/cr/minimal/backup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: infinispan.org/v2alpha1
+kind: Backup
```

Comment:
Not good idea to give common name for Backup/Restore instead of InfinispanBackup/InfinispanRestore
For example this operator also has Backup CRD name and in multioperator cluster doesn't make sense, i.e. `kubectl get backup` will return unpredictable results
https://operatorhub.io/operator/postgresql-operator-dev4devs-com

Reply:
I actually used the postrgres operator as inspiration when deciding between `InfinispanBackup` and `Backup` :smile:

As Backup is such a generic term, I don't think we or postgres can expect no conflicts, however in such a scenario you can use the FQN of the resource type, e.g: `kubectl get backups.infinispan.org`.

We also have the same issue with the "Cache" resource, as this is a generic name  that could also be used by a competing operator.

Reply:
Yes, i know, it's not a problem, but limiting use a short name for all users

Reply:
> Yes, i know, it's not a problem, but limiting use a short name for all users

Yes, `Cache` and `Batch` very common names and can confuse when will use by customers

Reply:
It's actually becoming pretty common even for default OpenShift resources to have same name differentiated by group. Try running `oc api-resources | sort` on clean installation. Having `infinispanBackup.infinispan.org` could be then considered as redundant naming?

Eg. images:
```
images                                                 config.openshift.io                   false        Image
images                                                 image.openshift.io                    false        Image
```

Reply:
Agree with you, @Crumby 
But if this CRD more system (operator) related and never been managed by users it can be, but if customer related it can occurs some confusions.

Reply:
> Having `infinispanBackup.infinispan.org` could be then considered as redundant naming?

That was my thoughts. I initially implemented it as `InfinispanBackup`, but it just seemed overly verbose in the commands and code.

Reply:
Ok, lets keep this name as defined. The logic much more important that name. I like Backup/Restore names, but care about users in complex operators environment.